zap/src/main/java/org/parosproxy/paros/core/scanner/ScannerParam.java

```diff
@@ -311,6 +312,14 @@ protected void parse() {
         }
     }
 
+    private void fixTypoForScannerAntiCSRF() {
```

Comment:
Add to `removeOldOptions` and rename it to `migrateOldOptions`.

Reply:
Do this in `migrateOldOptions`, there's no need to have it in its own method (simple statements and will be called just from one place).

Reply:
Bump. The commits can also be squashed.